lookerpy/apis/look_api.py

```diff
@@ -75,19 +76,20 @@ def all_looks(self, **kwargs):
             if key not in all_params:
                 raise TypeError(
                     "Got an unexpected keyword argument '%s'"
-                    " to method all_looks" % key
+                    " to method create_look_prefetch" % key
```

Comment:
this looks weird.

Reply:
This block follows the pattern of all the other methods in the class; the 'params = dir(self)' was a change I had made for debugging purposes.